.Documentation/articles/developer/14-validation.md

```diff
@@ -0,0 +1,36 @@
+# Validation
+
+**What do have to take into account to get validation properly working?**
+
+- The validation system can be enabled/disabled by switchting the CreatorProjectSettings entry `IsValidationEnabled`.
+- Validation only properly works with the correct scene opend in the unity editor.
```

Comment:
I would say that it also works having the incorrect scene/training course opened 🤔

Reply:
And it also shows a lot of reference errors which, in the correct scene, are working as intended.